app/src/main/java/org/mozilla/fenix/compose/ClickableSubstringLink.kt

```diff
@@ -45,7 +46,7 @@ fun ClickableSubstringLink(
         )
 
         addStyle(
-            SpanStyle(color = FirefoxTheme.colors.textAccent),
```

Comment:
if the accent color is being removed, should the `textColor` be applied to the entire `annotatedText` now?

Reply:
I was going to apply the color to the whole text directly, but I also had in mind (if this would end up being used in more places) to keep it more flexible. We could add both different coloring and underline, or use an enum to choose between them.

Reply:
Fair enough